.github/workflows/sanity.yaml

```diff
@@ -16,7 +16,5 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: '1.17'
-      - name: Vendor dependencies
-        run: go mod vendor
```

Comment:
One other alternative: use the official golangci-lint action vs. running through a root Makefile target. We could then house a custom .golangci.yml file so we have a consistent linting story when running locally vs. in CI.